### PR TITLE
Correctly unbind click events before rebinding

### PR DESCRIPTION
**Pull request by halfdan**
refs #1284

### CODE REVIEW

File: app/webroot/js/sentences_lists.menu.js [hunk 1]
```diff
@@ -125,5 +37,94 @@ $(document).ready(function() {
...
+            sentenceId = $(this).attr("data-sentence-id");
+            $("#addToList"+sentenceId).toggle();
+        });
+
+        // The sentence is added to the list after user has clicked
+        // the button. I was hesitating between this solution and
+        // having the sentence added to the list onChange, that is,
+        // directly after the selection in the <select>.
+        $(".validateButton").off();
+        $(".validateButton").click(function(){
```
Comment (halfdan) [inline app/webroot/js/sentences_lists.menu.js]: Had to move these back as they depend on `feedbackValid` being defined. Instead I simply `off` existing events and rebind them.

File: app/webroot/js/wall.reply.js [hunk 2]
```diff
@@ -84,73 +84,64 @@ $(document).ready(function(){
...
-            // that way we're sure replyFormInMessageId will not be set to -1
-            // before the end of saveMessage
-    }); 
-
+    $('.ajaxSubmit').click(function(){
+        saveMessage() ;
+        // this line is not in save message due to ajax's asynchronousity
+        // that way we're sure replyFormInMessageId will not be set to -1
+        // before the end of saveMessage
+    });
```
Comment (trang) [inline app/webroot/js/wall.reply.js]: For some reason the "Save" button (in reply form on the Wall) doesn't work anymore with this change.

Same issue with the "Cancel" button.

**Comment by halfdan** [comment]
Interesting, @trang, I'll check it out and make sure it works. It might be worth spending some time on proper unit/acceptance tests after we upgrade to a newer CakePHP version.

**Comment by trang** [comment]
@halfdan I'll merge your PR since it fixes #1389 and I'd like to deploy that fix on prod. I'll just revert the changes in `wall.reply.js`.

>  It might be worth spending some time on proper unit/acceptance tests after we upgrade to a newer CakePHP version.

Definitely.

**Pull request merged**